apps/examples/testcase/le_tc/kernel/tc_libc_misc.c

```diff
@@ -152,22 +161,173 @@ static void tc_libc_misc_crc16part(void)
 	uint16_t ret_chk;
 	uint8_t src_arr[1] = { VAL_100 };
 	uint16_t crc_16val = 0;
-	size_t length = 1;
 
 	/* Return value should be 100 as calculated by crc16part */
 
-	ret_chk = crc16part(src_arr, length, crc_16val);
+	ret_chk = crc16part(src_arr, LENGTH, crc_16val);
 	TC_ASSERT_EQ("crc16part", ret_chk, VAL_100);
 
 	/* Return value should be 65380 as calculated by crc16part */
 
 	crc_16val = VAL_255;
-	ret_chk = crc16part(src_arr, length, crc_16val);
+	ret_chk = crc16part(src_arr, LENGTH, crc_16val);
 	TC_ASSERT_EQ("crc16part", ret_chk, VAL_65380);
 
 	TC_SUCCESS_RESULT();
 }
 
+/**
+ * @fn                  :tc_libc_misc_crc16_ccitt_xmodem
+ * @brief               :Return a 16-bit CRC16-CCITT/Xmodem of the contents of the 'src' buffer, length 'len'
+ * @scenario            :Return a 16-bit CRC16-CCITT/Xmodem of the contents of the 'src' buffer, length 'len'
+ * @API's covered       :crc16_ccitt_xmodem
+ * @Preconditions       :None
+ * @Postconditions      :None
+ * @Return              :void
+ */
+static void tc_libc_misc_crc16_ccitt_xmodem(void)
+{
+	uint16_t ret_chk;
+	uint8_t src_arr[ARR_NUM] = { VAL_50, VAL_100 };
+	uint16_t res_arr[ARR_NUM] = { VAL_5649, VAL_11298 };
```

Comment:
The `res_arr` has VAL_XXX. But is this readable? Is 5649 value itself important? or Is result value of 50 hashed important?

Reply:
As you mentioned, VAL_XXX is not readable and the value is not important itself. I just followed the existing code, but if I have to change it, I think it's better to use raw number, not defined value.